apps/web/app/actions/doers.ts

```diff
@@ -349,32 +380,10 @@ export const createMemory = async (input: {
     );
   }
 
-  try {
```

Comment:
not sure if removing this is a good idea, this existed to make sure that there's 'consistency' between the vector store and our document store. are we still making sure that the data is consistent? if yes, yeah removing it is a great idea

Reply:
This had to be moved to account for error response form the /api/add endpoint ( the error related to the maximum chunk size ) and yeah the vector store and d1 store should be consistent 

But there is an edge case here and in main, if one fails and the other one succeeds then they won't be consistent........right?

Reply:
Yeah, we gotta make sure that doesn't happen. If it does maybe we should call the delete endpoints to get rid of them?

Reply:
Yes that or like a exponential back off retry for the failed db call, if that fails to then call the delete endpoing?

Reply:
Yes that would be absolutely amazing

Reply:
we don't _need_ retries or backoffs, it's a good to have, as long as we just ensure both are the same